src/Cubotino_T_display.py

```diff
@@ -0,0 +1,113 @@
+        font2 = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", fs2)  # font and size for second text row
+        disp_img = Image.new('RGB', (self.disp_w, self.disp_h), color=(0, 0, 0)) 
+        disp_draw = ImageDraw.Draw(disp_img)
+        disp_draw.text((x1, y1), r1, font=font1, fill=(255, 255, 255))    # first text row start coordinate, text, font, white color
+        disp_draw.text((x2, y2), r2, font=font2, fill=(255, 255, 255))    # second text row start coordinate, text, font, white color
+        self.disp.display(disp_img)                                       # image is plot to the display
+
+    def display_progress_bar(self, percent):
+        """ Function to print a progress bar on the display."""
+
+        # percent value printed as text 
+        fs = 40                 # font size
+        font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", fs)     # font and its size
+        text_x = int(self.disp_w/2 - (fs*len(str(percent))+1)/2)                   # x coordinate for the text starting location         
+        text_y = 15                                                           # y coordinate for the text starting location
+        disp_img = Image.new('RGB', (self.disp_w, self.disp_h), color=(0, 0, 0)) 
+        disp_draw = ImageDraw.Draw(disp_img)
+        disp_draw.text((text_x, text_y), str(percent)+'%', font=font, fill=(255, 255, 255))    # text with percent value
+        
+        # percent value printed as progress bar filling 
+        x = 15                  # x coordinate for the bar starting location
+        y = 60                  # y coordinate for the bar starting location
+        gap = 3                 # gap in pixels between the outer border and inner filling (even value is preferable) 
+        barWidth = 35           # width of the bar, in pixels
+        barLength = 135         # lenght of the bar, in pixels
+        filledPixels = int( x+gap +(barLength-2*gap)*percent/100)  # bar filling length, as function of the percent
+        disp_draw.rectangle((x, y, x+barLength, y+barWidth), outline="white", fill=(0,0,0))     # outer bar border
+        disp_draw.rectangle((x+gap, y+gap, filledPixels-1 , y+barWidth-gap), fill=(255,255,255)) # bar filling
+        
+        self.disp.display(disp_img) # image is plotted to the display
+
+
+    def show_cubotino(self):
+        """ Shows the Cubotino logo on the display."""
+                
+        image = Image.open("Cubotino_T_Logo_265x212_BW.jpg")  # opens the CUBOTino logo image (jpg file, converted and saved from pdf file)
+        image = image.resize((self.disp_w, self.disp_h))                # resizes the image to match the display.
+        self.disp.display(image)                                   # draws the image on the display hardware.
+        
+display = Display()
+
```

Comment:
if __name__ == "__main__":
    import time
    
    display.set_backlight(0)
    display.set_backlight(1)
    
    print("\nDisplay test for 20 seconds, started")
    print("Display shows rectangles, text and Cubotino logo")
    
    for i in range(5):
        display.test()
        time.sleep(2)
        display.show_cubotino()
        time.sleep(2)
    
    display.set_backlight(0)
    print("Display test finished\n")